packages/css-prettifier/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@emotion/css-prettifier",
```

Comment:
I'm not super fond of this name - it seems super close to `@emotion/css` but yet it doesn't have to do anything directly with it.

Reply:
I think it's fine. For mostly internal packages, I don't think it matters much